.github/workflows/integration-test.yml

```diff
@@ -13,81 +13,83 @@ jobs:
           mkdir -p plugins
       
-      - name: Checkout security kibana plugin
+      - name: Checkout OpenSearch Dashboard Security plugin
         uses: actions/checkout@v2
         with:
-          path: kibana/plugins/opendistro_security
+          path: OpenSearch-Dashboards/plugins/security-dashboards-plugin
           ref: ${{ github.ref }}
-      
-      - name: Get Kibana version
-        id: kbn_version
+
+      - name: Check OpenSearch Running
+        continue-on-error: true
+        run: curl -XGET https://localhost:9200 -u 'admin:admin' -k
+
+      - name: Get OpenSearch Dashboards version
+        id: osd_version
         run: |
-          echo "::set-output name=kbn_version::$(jq -r '.kibana.version' ./kibana/plugins/opendistro_security/package.json)"
+          echo "::set-output name=osd_version::$(jq -r '.opensearchDashboards.version' ./OpenSearch-Dashboards/plugins/security-dashboards-plugin/package.json)"
       
-      - name: Get Opendistro version
+      - name: Get OpenSearch version
         id: opendistro_version
         run: |
-          echo "::set-output name=opendistro_version::$(jq -r '.version' ./kibana/plugins/opendistro_security/package.json | sed 's/[0-9]*$/0/')"
-
-      - name: Run elasticsearch with plugin
-        run: |
-          wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-${{ steps.kbn_version.outputs.kbn_version }}-linux-x86_64.tar.gz
-          tar -xzf elasticsearch-oss-${{ steps.kbn_version.outputs.kbn_version }}-linux-x86_64.tar.gz
-          cd elasticsearch-${{ steps.kbn_version.outputs.kbn_version }}/
-          bin/elasticsearch-plugin install -b https://d3g5vo6xdbdb9a.cloudfront.net/downloads/elasticsearch-plugins/opendistro-security/opendistro-security-${{ steps.opendistro_version.outputs.opendistro_version }}.zip
-          bash plugins/opendistro_security/tools/install_demo_configuration.sh -y -i
-          echo 'opendistro_security.unsupported.restapi.allow_securityconfig_modification: true' >> config/elasticsearch.yml
-          echo $(curl -s -o /dev/null -w ''%{http_code}'' https://localhost:9200 -u admin:admin -k)
-          bin/elasticsearch &
-          sleep 30
-          timeout 900 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' https://localhost:9200 -u admin:admin -k)" != "200" ]]; do echo "ping ElasticSearch..."; sleep 5; done'
+          echo "::set-output name=opendistro_version::$(jq -r '.version' ./OpenSearch-Dashboards/plugins/security-dashboards-plugin/package.json | sed 's/[0-9]*$/0/')"
       
-      - name: Use kibana release commit
+      - name: Use OpenSearch Dashboards release commit
         run: |
-          cd ./kibana
-          git checkout tags/v${{ steps.kbn_version.outputs.kbn_version }} -b v${{ steps.kbn_version.outputs.kbn_version }}
+          cd ./OpenSearch-Dashboards
+          # git checkout tags/v${{ steps.osd_version.outputs.osd_version }} -b v${{ steps.osd_version.outputs.osd_version }}
```

Comment:
why commented out?

Reply:
this is for getting latest commit on dashboards. they may be adding changes that aren't captured by tags

Reply:
The latest commit is actually not as stable as tags. Tags got cut when some bugs are found and fixed. So tags are actually considered as a stable version, which is why tags are provided for Integ Tests and Unit Tests for plugins. This the case on OpenSearch. Is Dashboards different?